article/app/controllers/ArticleController.scala

```diff
@@ -89,7 +89,10 @@ class ArticleController(
     List(("html", views.html.fragments.articleBody(article))) ++ contentFieldsJson
   }
 
-  private def getGuuiJson(article: ArticlePage, blocks: Blocks)(implicit request: RequestHeader): String = {
+  /**
+    * Returns a JSON representation of the payload that's sent to DCR when rendering the Article.
+    */
+  private def getDCRJson(article: ArticlePage, blocks: Blocks)(implicit request: RequestHeader): String = {
```

Comment:
I don't think this function is specifically only rendering DCR? Seems like it also renders email html & 'just regular' JSON as well?

Reply:
Are you able to spell out how it does that?

This particular method looks to me like it will always return a JSON representation of `DotcomRenderingDataModel`?

https://github.com/guardian/frontend/blob/1043b49dd7e9e238c5522ae70a0a518ca4f00aa8/article/app/controllers/ArticleController.scala#L100-L103

Reply:
Github has fooled me! I had kept scrolling & missed that the `getDCRJson` function has ended & `render` function has begun! My bad, please ignore

Reply:
ah I see, no worries! yeah i've been caught out by the github UI on that sort of thing before too!